client/trigger_dockerfile_lint.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+import beanstalkc
+import json
+import os
+import sys
+
+if __name__ == "__main__":
+    name_space = sys.argv[1]
+    git_path = sys.argv[2]
+    filename = sys.argv[3]
+    notify_email = sys.argv[4]
+
+    # having '/' in a value used in os.path.join generates unexpected paths
+    if git_path.startswith("/"):
+        git_path = git_path[1:]
+
+    if filename.startswith("/"):
+        filename = filename[1:]
+
+    lint_job_data = {}
+
+    with open(os.path.join(os.environ['DOCKERFILE_DIR'], git_path, filename)) \
+    as f:
+        dockerfile = f.read()
+
+    bs = beanstalkc.Connection(host="openshift")
+    bs.use("start_lint")
```

Comment:
The job needs to be put master tube with action `start_lint` and in `worker_dispatcher.py` read the action and route the job to tube `start_lint` tube, then `worker_dockerfile_lint.py` will read to the tube. This way you are putting the right from this python script to `start_lint` job and `worker_dockerfile_lint.py` is reading it, this way the `worker_dispatcher.py` is bypassed. We should put `worker_dispatcher.py` is stop-relay point for all the jobs and have it as router.